src/testTrackConfig.js

```diff
@@ -1,5 +1,6 @@
 var TestTrackConfig = (function() { // jshint ignore:line
-    var config,
+    var DEFAULT_VISITOR_COOKIE_NAME = 'tt_visitor_id',
```

Comment:
Should this be the same as the name here: https://github.com/Betterment/test_track_js_client/pull/5/files#diff-930fd6f2d66487985b558ba323f39fe8R164 ?